src/containers/shared/components/DomainLink.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { decodeHex } from '../transactionUtils'
+
+export interface Props {
+  className?: string
+  decode?: boolean
+  domain: string
+}
+
+const DomainLink = (props: Props) => {
```

Comment:
maybe this should be called `ExternalLink` for a slightly more general name?

Reply:
We could have ExternalLink and have DomainLink use that.  The DomainLink component would then handle the decode and adding https and pass in the className of "domain".  External link would then handle rel and target.